lib/internal/http2/core.js

```diff
@@ -3189,6 +3190,10 @@ class Http2Server extends NETServer {
     validateSettings(settings);
     this[kOptions].settings = { ...this[kOptions].settings, ...settings };
   }
+
+  async [SymbolAsyncDispose]() {
+    return promisify(super.close).call(this);
```

Comment:
FunctionPrototypeCall primordial

Reply:
@benjamingr is this relevant even though the function we `.call` on is from `internal/util`?

Reply:
yes, since `Function.prototype.call` can be tamperd